src/test/java/eu/yals/test/TestUtils.java

```diff
@@ -26,143 +26,195 @@
+        return formatter.format(date);
     }
-    for (String testBrowser : testBrowsers) {
-      TestApp.Browser browser;
-      try {
-        browser = TestApp.Browser.valueOf(testBrowser.trim().toUpperCase());
-        browsers.add(browser);
-      } catch (IllegalArgumentException | NullPointerException e) {
-        log.error(String.format("Browser '%s' is not supported. Skipping...", testBrowser), e);
-      }
+
+    /**
+     * List of browser to run tests with.
+     *
+     * @return list of {@link TestApp.Browser}
+     */
+    public static List<TestApp.Browser> getTestBrowsers() {
+        List<TestApp.Browser> browsers = new ArrayList<>(1);
+
+        String testBrowsersProp = System.getProperty(TestApp.Properties.TEST_BROWSERS, "");
+        if (StringUtils.isBlank(testBrowsersProp)) {
+            return defaultBrowsers();
+        }
+
+        String[] testBrowsers = testBrowsersProp.trim().split(",");
+        if (testBrowsers.length <= 0) {
+            return defaultBrowsers();
+        }
+        for (String testBrowser : testBrowsers) {
+            TestApp.Browser browser;
+            try {
+                browser = TestApp.Browser.valueOf(testBrowser.trim().toUpperCase());
+                browsers.add(browser);
+            } catch (IllegalArgumentException | NullPointerException e) {
+                log.error(String.format("Browser '%s' is not supported. Skipping...", testBrowser), e);
+            }
+        }
+        if (browsers.isEmpty()) browsers = defaultBrowsers();
+        return browsers;
     }
-    if (browsers.isEmpty()) browsers = defaultBrowsers();
-    return browsers;
-  }
-
-  private static List<TestApp.Browser> defaultBrowsers() {
-    return Collections.singletonList(TestApp.Browser.CHROME);
-  }
-
-  /**
-   * Following needed because in may contain something like 'application/json;encoding=UTF8'
-   *
-   * @param contentType Content-Type header like 'application/json;encoding=UTF8'
-   * @return string which contains content type without encoding
-   */
-  private static String extractMime(String contentType) {
-    assertNotNull(contentType);
-
-    String[] contentTypeParts = contentType.split(";");
-    if (contentTypeParts.length > 1) {
-      return contentTypeParts[0];
-    } else {
-      return contentType;
+
+    private static List<TestApp.Browser> defaultBrowsers() {
+        return Collections.singletonList(TestApp.Browser.CHROME);
     }
-  }
-
-  private static boolean isValidErrorJson(HttpResponse<String> result) {
-    String body = result.getBody();
-    try {
-      ErrorJson errorJson = AppUtils.GSON.fromJson(body, ErrorJson.class);
-      assertNotNull(errorJson);
-      boolean hasNotEmptyErrorField = errorJson.getError() != null;
-      boolean hasNotEmptyErrorsField = errorJson.getErrors() != null;
-
-      return hasNotEmptyErrorField || (hasNotEmptyErrorsField && errorJson.getErrors().size() > 0);
-    } catch (Exception e) {
-      return false;
+
+    /**
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [First sentence should end with a period.](https://app.codacy.com/gh/yadevee/yals/pullRequest?prid=5409395)